infrastructure/quick-deploy/aws/all/variables.tf

```diff
@@ -439,6 +439,18 @@ variable "cloudwatch" {
   default = {}
 }
 
+variable "s3" {
+  description = "S3 bucket for logs"
+  type = object({
+    enabled = optional(bool, true)
+    name    = optional(string, "armonik-logs")
+    region  = optional(string, "eu-west-3")
```

Comment:
I don't think the region should be part of the default values.

Reply:
we have to give the name of the bucket S3 and its region as input to fluent bit

Reply:
I understand why you need the region parameter, but I'm not sure we should give a default value. After all, this is a valid assumption only for us.